Cargo.toml

```
@@ -8,4 +8,12 @@ license = "Apache-2.0"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-cesride = "0.1.3"
+cesride = { git = "https://github.com/DSRCorporation/cesride.git" }
+nom = "5"
+num-derive = "0.3.3"
+num-traits = "0.2"
+rmp-serde = "0.15"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
```

```suggestion
serde_json = { version = "1.0", features = ["preserve_order"] }
```

In this context I believe this is a requirement.

---

https://docs.rs/crate/serde_json/1.0.93/features